bindings/pyroot/pythonizations/python/ROOT/_numbadeclare.py

```diff
@@ -1,5 +1,6 @@
 from cppyy import gbl as gbl_namespace
 
+import sys
```

Comment:
I would move the import of `sys` just after `re`. The rational is that we want to import as little as possible at import time of ROOT. `sys` is not a big deal actually, though for correctness I would move it in the function.

@etejedor Do you agree?

Reply:
Yes ok!

Reply:
Better?

Reply:
Perfect, thanks! :)